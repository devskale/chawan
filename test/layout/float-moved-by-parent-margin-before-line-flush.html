<div style="width: 20ch; display: flow-root">
<div style="margin-top: 1em">
<div style="float: right">
<div style="height: 4em; background: red; display: inline-block">right</div>
</div>
test<br>
test
<div style="margin-top: 1em">
test test test test
