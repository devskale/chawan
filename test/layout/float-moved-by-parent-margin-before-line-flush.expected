
test           [48;2;255;0;0mright[49m
test           [48;2;255;0;0m     [49m
               [48;2;255;0;0m     [49m
test test test [48;2;255;0;0m     [49m
test
